azure-pipelines.yml

```diff
@@ -4,13 +4,40 @@
 
 name: $(TeamProject)_$(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
 
-variables:
-  MyRunNumber:  $[counter('MyRunNumber', 52201)]
-
+parameters:
   # Set this to 'true' to build the libtorch-* packages as part of main branch CI and
   # push them to the artifacts feed of the Azure CI project
-  BuildLibTorchPackages: false
-  PushPackagesToFeed: false
+  - name: BuildLibTorchPackages
+    type: boolean
+    default: false
+  - name: PushPackagesToFeed
+    type: boolean
+    default: false
+
+variables:
+  MyRunNumber:  $[counter('MyRunNumber', 52201)]
+  LinuxPrepScript: |
+    ldd --version && (/sbin/ldconfig -p | grep stdc++) && (strings /usr/lib/x86_64-linux-gnu/libstdc++.so.6 | grep LIBCXX)
+    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys DE19EB17684BA42D
+    sudo apt-get -y update
+    sudo apt-get -y install cmake git libunwind8 curl libomp-dev libomp5
+    wget https://packages.microsoft.com/config/ubuntu/22.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+    sudo dpkg --purge packages-microsoft-prod && sudo dpkg -i packages-microsoft-prod.deb
+    sudo apt-get update; sudo apt-get install -y apt-transport-https && sudo apt-get update
+    ldd --version && (/sbin/ldconfig -p | grep stdc++) && (strings /usr/lib/x86_64-linux-gnu/libstdc++.so.6 | grep LIBCXX)
+
+    # Install Miniconda
```

Comment:
Configure cmake/clang/gcc to match the version used by `win`/`macos` building machine using `miniconda` environment

Reply:
No longer applied

Reply:
Not sure I understand what you are saying. What is no longer applied?

Reply:
I mean the change of installing conda environment in pipeline no longer applies to the latest commit